x-pack/plugins/observability_solution/apm/public/components/shared/add_data_buttons/buttons.tsx

```diff
@@ -34,22 +34,24 @@ export const collectServiceLogs = {
   link: '/app/observabilityOnboarding/customLogs/?category=logs',
```

Comment:
Are we keeping this hardcoded? or is it better to replace it in another PR?

Reply:
I looked into how to use the locator but didn’t quite figure it out. Rather than hold up the PR, I’ll leave this improvement for a future PR.